controllers/whoToFollow/whoToFollow.js

```diff
@@ -122,13 +131,34 @@ module.exports = async (req, res) => {
       if (tempUsers.length > 0) {
         result.push({
           ...location,
-          viewtype: 'labellocation'
+          viewtype: 'labellocation',
+          users: tempUsers
         });
+      }
+    }
 
-        result.push(...tempUsers);
+    //Other topics
+    const tempUsers = [];
+    for (const indexUserOtherTopic in userOtherTopicFollower) {
+      const userOtherTopic = userOtherTopicFollower[indexUserOtherTopic];
+      delete userOtherTopic.human_id;
+      if (userOtherTopic.user_id !== process.env.BETTER_ADMIN_ID) {
+        tempUsers.push({
+          ...userOtherTopic,
+          viewtype: 'user'
+        });
       }
     }
 
+    if (tempUsers.length > 0) {
+      result.push({
+        viewtype: 'labelothers',
+        name: 'Other popular members',
+        users: tempUsers
+      });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code is deleting the `human_id` field from the `userOtherTopic` object. Similar to the previous comments, if `userOtherTopicFollower` is a reference to an object stored elsewhere in your application, this could unintentionally affect other parts of your application that rely on the `human_id` property. Consider creating a copy of the object before modifying it.
```diff
-       const userOtherTopic = userOtherTopicFollower[indexUserOtherTopic];
-       delete userOtherTopic.human_id;
+       const userOtherTopic = {...userOtherTopicFollower[indexUserOtherTopic]};
+       delete userOtherTopic.human_id;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->